go.mod

```diff
@@ -2,8 +2,6 @@ module github.com/CiscoDevNet/terraform-provider-aci/v2
 
 go 1.21
 
-toolchain go1.21.5
```

Comment:
Is this meant to be removed?

Reply:
i changed this in the recent commit.

Reply:
Did you intend to remove this though because I don't think its meant to be removed. Unless I am missing completely something.

Reply:
it is not there in current master

Reply:
I don't think this was fixed.
Can you please use `go mod tidy`?

Reply:
fixed